src/registrar/models/domain_information.py

```diff
@@ -0,0 +1,258 @@
+        help_text="Type of work of the organization",
+    )
+
+    more_organization_information = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Further information about the government organization",
+    )
+
+    authorizing_official = models.ForeignKey(
+        "registrar.Contact",
+        null=True,
+        blank=True,
+        related_name="information_authorizing_official",
+        on_delete=models.PROTECT,
+    )
+
+    domain = models.OneToOneField(
+        "registrar.Domain",
+        on_delete=models.PROTECT,
+        blank=True,
+        null=True,
+        # Access this information via Domain as "domain.info"
+        related_name="domain_info",
+        help_text="Domain to which this information belongs",
+    )
+    alternative_domains = models.ManyToManyField(
+        "registrar.Website",
+        blank=True,
+        related_name="alternatives+",
+    )
+
+    # This is the contact information provided by the applicant. The
+    # application user who created it is in the `creator` field.
+    submitter = models.ForeignKey(
+        "registrar.Contact",
+        null=True,
+        blank=True,
+        related_name="submitted_applications_information",
+        on_delete=models.PROTECT,
+    )
+
+    purpose = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Purpose of your domain",
+    )
+
+    other_contacts = models.ManyToManyField(
+        "registrar.Contact",
+        blank=True,
+        related_name="contact_applications_information",
+    )
+
+    no_other_contacts_rationale = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Reason for listing no additional contacts",
+    )
+
+    anything_else = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Anything else we should know?",
+    )
+
+    is_policy_acknowledged = models.BooleanField(
+        null=True,
+        blank=True,
+        help_text="Acknowledged .gov acceptable use policy",
+    )
+    security_email = models.EmailField(
+        max_length=320,
+        null=True,
+        blank=True,
+        help_text="Security email for public use",
+    )
+
+    def __str__(self):
+        try:
+            if self.domain and self.domain.name:
+                return self.domain.name
+            else:
+                return f"application created by {self.creator}"
+        except Exception:
+            return ""
+
+    @classmethod
+    def create_from_da(cls, domain_application):
+        """Takes in a DomainApplication dict and converts it into DomainInformation"""
+        da_dict = domain_application.to_dict()
+        # remove the id so one can be assinged on creation
+        da_id = da_dict.pop("id")
+        # check if we have a record that corresponds with the domain
+        # application, if so short circuit the create
+        domain_info = cls.objects.filter(domain_application__id=da_id).first()
+        if domain_info:
```

Comment:
This is more of check to see if this record exist. see comment on line 225-226. If it see that it exist, it will "skip" the whole creation process. Without that, it will cause duplicates record of Domain Information for the same domain.